Sources/RayTracer/RayTracer.Imaging/Png/PngColorType.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace RayTracer.Imaging.Png;
+
+[Flags]
+public enum PngColorType : byte
+{
+    None = 0,
```

Comment:
0 is rather Grayscale (not None).

Also, values in [Flags] enum should be set via bit shifting:

```
[Flags]
enum PngColorType : byte
{
    Greyscale = 0,
    PalletteUsed = 1 << 0,
    ColorUsed = 1 << 1,
    ACUsed = 1 << 2
}
```